.travis.yml

```diff
@@ -34,6 +34,9 @@ script:
   - python setup.py sdist >/dev/null
   - echo "check_stable=False" >pysal/config.py
   - travis_wait 45 nosetests --verbose --with-coverage --cover-package=pysal;
+  - nosestests --verbose --with-coverage pysal/dynamics
+  - nosestests --verbose --with-coverage pysal/explore
```

Comment:
I would like to check if `pytest` does this right.

Reply:
Worth a shot. If pytest isn't the solution, we can always go with the explicit approach here.